NOTICE-3RD-PARTY-CONTENT.md

```diff
@@ -57,4 +57,4 @@
 |irongut/CodeCoverageSummary|v1.2.0|MIT License|
 |marocchino/sticky-pull-request-comment|v2|MIT License|
 |pre-commit/action|v3.0.0|MIT License|
-|softprops/action-gh-release|v1|MIT License|
\ No newline at end of file
+|softprops/action-gh-release|v1|MIT License|
```

Comment:
It seems there is a whitespace change here, I suggest we undo it.

Reply:
a newline has to be added at the end of the file, otherwise the license finder will fail...